engine/class_modules/sc_shaman.cpp

```diff
@@ -10952,8 +10959,21 @@ void shaman_t::init_spells()
 
   // Constants
   constant.mul_matching_gear = spec.mail_specialization->effectN( 1 ).percent();
-  constant.mul_lightning_rod = find_spell( 210689 )->effectN( 2 ).percent() +
-                               spec.enhancement_shaman->effectN( 27 ).percent();
+
+  constant.mul_lightning_rod = find_spell( 210689 )->effectN( 2 ).percent();
+
+  // On 11.0.5 PTR, Lightning Rod receives a multiplicative damage buff from Charged Conduit
+  if ( is_ptr() )
+  {
+      if ( talent.charged_conduit->ok() )
+      {
+            constant.mul_lightning_rod *= ( 1 + talent.charged_conduit->effectN( 2 ).percent() );
+      }
+  }
+
+  // Add enhancement Lightning Rod modifier
+  constant.mul_lightning_rod += spec.enhancement_shaman->effectN( 27 ).percent();
+
```

Comment:
This is ugly af but without refactoring the whole of LRod, I'm not sure of a nicer way to do this.